### PR TITLE
Patch test runner

### PR DESCRIPTION
**Pull request by jnewman**

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -54,6 +55,7 @@ gulp.task('lint', ['eslint', 'jscs']);
 
 function build(config) {
   return () => webpackStream(config)
+    .pipe(replace('Object.defineProperty', '(function(t, k, d) {return t[k] = d.value;})'))
```
Comment (jnewman) [inline gulpfile.babel.js]: Otherwise IE8 gets really mad.

File: package.json [hunk 2]
```diff
@@ -67,9 +67,10 @@
     "gulp-header": "1.7.1",
     "gulp-jscs": "3.0.2",
     "gulp-rename": "1.2.2",
+    "gulp-replace": "0.5.4",
     "gulp-strip-debug": "1.1.0",
     "gulp-uglify": "1.5.3",
-    "jquery": "2.2.3",
+    "jquery": "1.11.3",
```
Comment (jnewman) [inline package.json]: 2.x doesn't support IE8, so the tests on sauce would never pass.

File: .gitignore [hunk 3]
```diff
@@ -13,3 +13,5 @@ npm-*.log
 
 # Generated files
 .idea/
+
+dist/
```
Comment (sethyates) [inline .gitignore]: I think this will break `npm publish`?

File: gulpfile.babel.js (hunk 1 above)
Comment (sethyates) [inline gulpfile.babel.js]: There's no babel transform for this?

File: .gitignore (hunk 3 above)
Comment (jnewman) [inline .gitignore]: I checked and if `.npmignore

**Pull request merged**